lib/utils.ts

```diff
@@ -88,12 +88,13 @@ export const navLinks = [
   { title: "People", path: "/people" },
   { title: "Projects", path: "/projects" },
   { title: "Releases", path: "/releases" },
+  { title: "Discussion", path: "/discussion" },
```

Comment:
Update the [`sitemap.ts`](https://github.com/coronasafe/leaderboard/blob/main/app/sitemap.ts) too to include this new page